packages/scheduler/src/WrappedTarget.ts

```diff
@@ -214,7 +214,7 @@ export class WrappedTarget implements TargetRun {
               worker.postMessage({ type: "hash", hash });
             });
           } else if (this.options.onMessage) {
-            this.options.onMessage(data, worker.postMessage);
+            this.options.onMessage(data, worker.postMessage.bind(worker));
```

Comment:
If you're needing to bind(), then you should probably create that once per "setup" - create a const before that line 209.

Reply:
Oh, you are right. Otherwise it'll be created on every message :(

Reply:
Fixed